pkg/api/handlers/compat/containers_stats.go

```diff
@@ -22,14 +22,19 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
 	decoder := r.Context().Value("decoder").(*schema.Decoder)
 
 	query := struct {
-		Stream bool `schema:"stream"`
+		Stream  bool `schema:"stream"`
+		oneShot bool `schema:"one-shot"` //added schema for one shot
```

Comment:
```suggestion
		OneShot bool `schema:"one-shot"` //added schema for one shot
```
Field needs to be exported.